frontend/public/components/overview/index.tsx

```diff
@@ -1027,6 +1052,12 @@ const Overview_: React.SFC<OverviewProps> = (({mock, namespace, selectedItem, se
       namespace,
       prop: 'statefulSets',
     },
+    {
+      isList: true,
+      kind: referenceForModel(VirtualMachineModel),
+      namespace,
+      prop: 'virtualMachines',
```

Comment:
Good point.

What I can do is follow the usual flag-connection strategy: connect the component to KubeVirt feature via `connectToFlags(FLAGS.KUBEVIRT)`, if `flagPending(kubevirtFlag)` then render `null`, otherwise render the current output with `resources` array either having or not having the VM-specific entry.

Does that sound good to you?

Reply:
In this specific case, I wouldn't worry about the pending check so we don't block on rendering the overview when kubevirt isn't installed. I'd treat `undefined ` and `false` the same. That way the rest of the resources load, then virtual machines are filled in when the flag changes to true.

Reply:
> I'd treat `undefined ` and `false` the same.

Agreed, makes sense.